Terminal.Gui/ViewBase/View.Layout.cs

```diff
@@ -635,7 +638,7 @@ internal void LayoutSubViews ()
 
         List<View> redo = new ();
 
-        foreach (View v in ordered)
+        foreach (View v in ordered.Snapshot ())
```

Comment:
Snapshot is very expensive. Does it rally need to be used here (and the other places where you've used it)?

Reply:
As you can see this is the only branch that doesn't fail in the integration unit tests.

Reply:
I'm only using Snapshot in the places where was causing the error that the collection was changed, mainly in FluentTests.